pombola/core/static/js/functions.js

```diff
@@ -198,4 +198,59 @@ $(function () {
     e.preventDefault();
     location.href = "/search?q=" + escape($('#id_q,#loc').first().val());
   });
+
+  const showMessageToolsId = $("#show-message-tools-toggle")
+  const messagesContainer = $("#message-tips-container")
+
+  // hide element initially if page width is smaller than 991px
+  if (window.innerWidth < 991) {
+    messagesContainer.hide()
+  }
+
+  // check for window resize actions and toggle show message tools
+  window.addEventListener('resize', function(event){
+    if (window.innerWidth < 991) {
+      messagesContainer.hide()
+    } else {
+      messagesContainer.show()
+    }
+  });
```

Comment:
In future use a reusable function rather than having the `window.innerWidth < 991` logic in multiple places.

Reply:
noted